tailwind.config.js

```diff
@@ -130,13 +130,34 @@ module.exports = {
         },
       },
       boxShadow: {
-        't-sm': '0 -1px 2px 0 rgba(0, 0, 0, 0.05)',
-        't': '0 -1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1);',
-        't-md': '0 -4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06)',
-        't-lg': '0 -10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)',
-        't-xl': '0 -20px 25px -5px rgba(0, 0, 0, 0.1), 0 10px 10px -5px rgba(0, 0, 0, 0.04)',
-        't-2xl': '0 -25px 50px -12px rgba(0, 0, 0, 0.25)',
-        't-3xl': '0 -35px 60px -15px rgba(0, 0, 0, 0.3)',
+        "t-sm": "0 -1px 2px 0 rgba(0, 0, 0, 0.05)",
+        t: "0 -1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1);",
+        "t-md": "0 -4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06)",
+        "t-lg": "0 -10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)",
+        "t-xl": "0 -20px 25px -5px rgba(0, 0, 0, 0.1), 0 10px 10px -5px rgba(0, 0, 0, 0.04)",
+        "t-2xl": "0 -25px 50px -12px rgba(0, 0, 0, 0.25)",
+        "t-3xl": "0 -35px 60px -15px rgba(0, 0, 0, 0.3)",
+      },
+      fontSize: {
+        15: [
+          "0.938rem", //15px
+          {
+            lineHeight: "1.25rem",
+          },
+        ],
+        19: [
+          "1.188rem", //19px
+          {
+            lineHeight: "1.5rem",
+          },
+        ],
+      },
+      maxHeight: {
+        "screen-60": "60vh",
+        "screen-75": "75vh",
+      },
+      margin: {
+        4.5: "1.125rem", //18px

```

Comment:
Need to recheck the design. Do we really need this margin? Can we use 16 or 20px instead?

Reply:
we have 3.5 by default, so why don't we also have 4.5 by default.

Reply:
this margin is used to prevent jerking when an error message appears